local-deploy.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Version Control Center - The following variables are updated to specify the dependency versions Fortis is currentl running on.
+#       For Example: minJava is the minimum Java version required at time of publish. Currently it is 1.7.0 which this script removes decimals
+#       for arithmetic analysis when validating your machine's Jdk version. - Updated 10.25.2017
+minJava=170
+minMaven=300
+minNode=400
+minNpm=300
+minScala=270
+minCassandra=300
+minSpark=2
+#----------------------------------------------------------------------------------
+echo "Checking your current system for Fortis compatability:"
+echo
+echo "Validating JAVA version and jdk...........................-"
+
+if type -p java; then
+    echo found JAVA executable in PATH
```

Comment:
Nit: might as well quote the echoed string to make this look more expected and to be consistent in all echo call sites: `echo "found JAVA executable in PATH"`

Reply:
Same elsewhere in this file.